### PR TITLE
#211 (UNLV-FIA Homepage) Change Group to Club Example (Based On main)

### PR DESCRIPTION
**Pull request by Kelvyn-Meyers**
This pull request is for issue #211.

## CODE CHANGES
1. Changed UNLV-FIA Group to UNLV-FIA Club
2. Added a smiley face to the header

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,8 +37,8 @@
 <div class="pimg_1">
   <div class="ptext">
     <div class="border">
-        <h1>UNLV-FIA</h1>
-        Group
+        <h1>UNLV-FIA ;)</h1>
```
Comment (ballec6) [inline index.html:40]: why smiley?

**Review by ballec6 — CHANGES_REQUESTED** [review]
Please see my requested changes below!

**Review by Rosadc1 — APPROVED** [review]
Good job!

**Comment by Kelvyn-Meyers** [comment]
Closing this example pull request.

**Pull request closed without merging**